numpy/random/tests/test_random.py

```diff
@@ -440,6 +440,14 @@ def test_choice_return_shape(self):
         assert_equal(np.random.choice(6, s, replace=False, p=p).shape, s)
         assert_equal(np.random.choice(np.arange(6), s, replace=True).shape, s)
 
+        # Check zero-size
+        assert_equal(np.random.randint(0,0,(3,0,4)).shape, (3,0,4))
+        assert_equal(np.random.randint(0,-10,0).shape, (0,))
+        assert_equal(np.random.choice(0,0).shape, (0,))
+        assert_equal(np.random.choice([],(0,)).shape, (0,))
```

Comment:
nit: these tests would be clearer if the size/shape argument was passed by kwarg - too many zeros here

Reply:
fixed